components/StyledInputTags.js

```diff
@@ -183,9 +185,16 @@ const StyledInputTags = ({ suggestedTags, value, onChange, renderUpdatedTags, de
     }
   };
 
+  // Setup popper to close with next call useGlobalBlur
+  useGlobalBlur(popperRef, outside => {
+    if (outside) {
+      popperRef.current = null;
+    }
+  });
+
   // Close when clicking outside
   useGlobalBlur(wrapperRef, outside => {
-    if (outside) {
+    if (outside && !popperRef.current) {
       handleClose();
     }
   });
```

Comment:
Can we maybe merge these two, to have only one `useGlobalBlur`?

Reply:
Thanks for the feedback @Betree ! I'm not having luck replicating the current UX by merging into one `useGlobalBlur`. I'm stuck between getting the behavior of clicking outside of the `Popper`/`popperRef` to close the menu, in which case the `StyledTag` button does not toggle appropriately (Popper opens back up on mouse-up). Likewise, if I call `useGlobalBlur`, passing in the `wrapperRef`, the Popper closes immediately on click, preventing any sort of useful interaction. 

I've also tried, adding a toggle to the `AddTag` button to manage a state for a `isClosable` var (like what was done in `AdminExpenseStatusTag`, but this seems to only work some of the time. 

I'm happy to try any suggestions you have here. Thank you!